packages/react-components/src/components/CaptionsSettingModal.tsx

```diff
@@ -73,8 +73,8 @@ export const _CaptionsSettingModal = (props: _CaptionsSettingModalProps): JSX.El
   const theme = useTheme();
 
   const [selectedItem, setSelectedItem] = useState<IDropdownOption>({
-    key: currentSpokenLanguage !== '' ? currentSpokenLanguage : 'en-us',
-    text: currentSpokenLanguage !== '' ? currentSpokenLanguage : 'en-us'
+    key: currentSpokenLanguage && currentSpokenLanguage !== '' ? currentSpokenLanguage : 'en-us',
```

Comment:
is currentSpokenLanguage going to be undefined or empty string?

We are checking if its an empty string but we also just said it could be undefined. Can we determine if its only undefined or only an empty string but not potentially both?